server.js

```diff
@@ -49,15 +49,27 @@ app.post('/api/register', async (req, res) => {
   }
 });
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
-});
-
 // app.post('/api/achievement', async (req, res) => {
 //   console.log("It's alive!");
 //   res.status(200).json({ message: 'Achievement successful' });
 // });
 
+app.post('/api/attempt', async (req, res) => {
```

Comment:
What is being attempted here? 

Login attempt or game attempt?

Reply:
Great question, it's a user attempt at the attempting the game with various attempt details. This could be refactored.